framework/src/main/scala/org/sunbird/obsrv/core/model/SystemConfig.scala

```diff
@@ -1,13 +1,16 @@
 package org.sunbird.obsrv.core.model
 
+import org.sunbird.obsrv.core.util.SystemConfigSelector
+import org.sunbird.obsrv.core.model.Models.SystemSettings
 object SystemConfig {
+  def getSystemConfig(key: String): Option[SystemSettings] = {
+    SystemConfigSelector.getSystemConfigurations().find(config => config.objectKey == key)
+  }
```

Comment:
Modify it to read the property only instead of reading all configurations and selecting one property